templates/parachain/Cargo.toml

```diff
@@ -14,3 +14,12 @@ docify = { workspace = true }
 
 [features]
 generate-readme = []
+
+[profile.release]
```

Comment:
I think profiles need to be added to the workspace Cargo.toml?  
Or does it also work like this 🤔 ?

Reply:
I think it should go in here https://github.com/paritytech/polkadot-sdk-parachain-template/blob/master/Cargo.toml so that the runtime also gets this profile.